code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -282,12 +282,22 @@
 		if(!LH.target)
 			var/datum/objective/A = new
 			A.owner = user.mind
+			var/list/datum/team/teams = list()
+			for(var/datum/antagonist/antag in user.mind.antag_datums)
+				var/datum/team/T = antag.get_team()
+				if(T)
+					teams |= T
 			var/list/targets = list()
 			for(var/i in 0 to 3)
 				var/datum/mind/targeted =  A.find_target()//easy way, i dont feel like copy pasting that entire block of code
+				var/is_teammate = FALSE
+				for(var/datum/team/T in teams)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names